src/granite/transactions.cr

```diff
@@ -143,6 +159,11 @@ module Granite::Transactions
       true
     end
 
+
+    def save!
+      save || raise Granite::Transactions::TransactionError.new("Could not save #{self.name}")
```

Comment:
I like this "save || raise" format better than the previous "raise unless save", but as someone else said, the explicit prefix `Granite::Transactions` can be dropped from these.